### PR TITLE
Disallow unsupported reducers, operators and filters

### PR DESCRIPTION
**Pull request by aesedepece**
These changes derive from these issues:
- https://github.com/witnet/witnet-rust/issues/1458
- https://github.com/witnet/witnet-rust/issues/1459

### CODE REVIEW

**Comment by aesedepece** [comment]
This is missing fixing the tests.

**Comment by aesedepece** [comment]
This is ready for review

**Review by Tommytrg — APPROVED** [review]

File: src/utils.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ export function getDefaultMirArgumentByType(type: MirArgumentType): MirArgument
     case MirArgumentType.Boolean:
       return true
     case MirArgumentType.FilterFunction:
-      return [Filter.lessThan, 0]
+      return [Filter.deviationStandard, 0]
```
Comment (mariocao) [inline src/utils.ts]:
```suggestion
      return [Filter.deviationStandard, 1.1]
```

**Review by mariocao — CHANGES_REQUESTED** [review]

**Comment by aesedepece** [comment]
Updated with @mariocao's suggestion!

**Review by mariocao — APPROVED** [review]

**Pull request merged**